### PR TITLE
input_filesuffix

### PR DESCRIPTION
**Pull request by anoukvlug**
Here is a PR to make it possible to use different file suffixes for the input climate files. This follows up on issue #301.

### CODE REVIEW

File: oggm/core/models/flowline.py [hunk 1]
```diff
@@ -1748,6 +1748,8 @@ def run_from_climate_data(gdir, ys=None, ye=None, filename='climate_monthly',
      filename : str
          name of the climate file, e.g. 'climate_monthly' (default) or
          'cesm_data'
+     input_filesuffix: str
+         filesuffix for the input file
```
Comment (fmaussion) [inline oggm/core/models/flowline.py]: Change to ``filesuffix for the input climate file``

File: oggm/core/models/massbalance.py [hunk 2]
```diff
@@ -173,8 +173,8 @@ def __init__(self, gdir, mu_star=None, bias=None, prcp_fac=None,
         filename : str, optional
             set to a different BASENAME if you want to use alternative climate
             data.
-        filesuffix : str
-            append a suffix to the filename (useful for model runs).
+        input_filesuffix : str
+            the file suffix of the input file
```
Comment (fmaussion) [inline oggm/core/models/massbalance.py]: Change to ``filesuffix for the input climate file``

File: oggm/core/models/massbalance.py [hunk 3]
```diff
@@ -201,7 +201,7 @@ def __init__(self, gdir, mu_star=None, bias=None, prcp_fac=None,
         self.temp_bias = 0.
 
         # Read file
-        fpath = gdir.get_filepath(filename, filesuffix=filesuffix)
+        fpath = gdir.get_filepath(filename, input_filesuffix=input_filesuffix)
```
Comment (fmaussion) [inline oggm/core/models/massbalance.py]: No need for a new keyword here: simply write ``fpath = gdir.get_filepath(filename, filesuffix=input_filesuffix)``

File: oggm/utils.py [hunk 4]
```diff
@@ -2367,7 +2367,8 @@ def copy_to_basedir(self, base_dir, setup='run'):
         else:
             raise ValueError('setup not understood: {}'.format(setup))
 
-    def get_filepath(self, filename, div_id=0, delete=False, filesuffix=''):
+    def get_filepath(self, filename, div_id=0, delete=False,
+                     input_filesuffix='', filesuffix=''):
```
Comment (fmaussion) [inline oggm/utils.py]: In accordance with my comment above, no need for a new keyword argument: ``utils.py`` can be left unchanged

**Comment by fmaussion** [comment]
Nevermind, I need to work on this so I'll do the changes now - thanks!

**Comment by anoukvlug** [comment]
Thank you @fmaussion!

**Pull request merged**